src/styles/header.scss

```diff
@@ -0,0 +1,60 @@
+.logo {
+  @include square($logoSize);
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    display: inline-block;
+    margin-left: 20px;
+
+    &:first-child {
+      height: 60px;
```

Comment:
```suggestion
```
do not hardcode height for first-child
instead set vertical paddings for link to make them on full height of navigation.
also it should fix tests

![Screenshot from 2023-05-24 12-04-14](https://github.com/mate-academy/layout_catalog/assets/33029864/f6d599f8-f958-4c39-978a-8a3529a9c965)